### PR TITLE
[LIVE-7271] Feature - Update Base Fee mutliplier for EIP1559 transactions

### PR DESCRIPTION
**Pull request by lambertkevin**
### 📝 Description

This PR is modifying the mulitiplier for the `baseFee` from `2` to `1.5` in order to reduce the visual impact on fees (at the expense of potentially having more users stuck with a transaction with not enough fees).
It also introduce an experimental env variable to make it customisable by everyone if needed.

### ⚠️ Impacted Currencies
- Ethereum
- Polygon
- Ethereum Goerli

### ❓ Context

- **Impacted projects**: `@ledgerhq/live-env` `@ledgerhq/live-common` `ledger-live-mobile` `ledger-live-desktop` 
- **Linked resource(s)**: https://ledgerhq.atlassian.net/browse/LIVE-7271 

### ✅ Checklist

- [ ] **Test coverage** // test coverage can only be the bot on this I guess as we don't have anything better already here
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

https://user-images.githubusercontent.com/44363395/234917680-09d46f10-5b13-4f22-a885-dba7601826db.mov

https://user-images.githubusercontent.com/44363395/234917732-2ed53405-7122-407d-9bd0-e8cf185d5b8b.mp4

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/screens/settings/sections/Experimental/ExperimentalFloat.tsx
Comment (lambertkevin) [inline apps/ledger-live-desktop/src/renderer/screens/settings/sections/Experimental/ExperimentalFloat.tsx:1]: I know this is ugly, it's a copy paste of the `ExperimentalInteger.tsx` with just a few tweaks 🙏

File: apps/ledger-live-mobile/src/families/ethereum/EthereumCustomFees/utils.tsx [hunk 1]
```diff
@@ -31,7 +31,9 @@ export const inferMaxFeeRange = (
     return defaultMaxFeePerGasRange;
 
   const { maxPriorityFeePerGas, nextBaseFeePerGas } = networkInfo;
-  const amplifiedBaseFee = nextBaseFeePerGas.times(2); // ensuring 6 blocks of base fee
+  const amplifiedBaseFee = nextBaseFeePerGas.times(
+    getEnv("EIP1559_BASE_FEE_MULTIPLIER"),
+  ); // ensuring 6 blocks of base fee
```
Comment (haammar-ledger) [inline apps/ledger-live-mobile/src/families/ethereum/EthereumCustomFees/utils.tsx:36]: Comment could be updated I guess, nbd (others as well)

**Review by haammar-ledger — APPROVED** [review]

File: apps/ledger-live-mobile/src/families/ethereum/EthereumCustomFees/utils.tsx (hunk 1 above)
Comment (lambertkevin) [reply apps/ledger-live-mobile/src/families/ethereum/EthereumCustomFees/utils.tsx:36]: Yup, missed this one, updating it 👍

Comment (lambertkevin) [reply apps/ledger-live-mobile/src/families/ethereum/EthereumCustomFees/utils.tsx]: Done

**Pull request merged**